### DISCUSSION TITLE
Opened files in external program get closed when leaving yazi

### DISCUSSION THREAD

**Discussion by a-torgovitsky** [sxyazi/yazi#1796] [1. Q&A] [answered]
I am running yazi in a tmux session like this:
```
kitty tmux new-session "yazi /home/at/"
```
I open a .pdf in the system viewer (evince for me) with yazi via `o`.
If I then quit yazi, the instance of the system viewer is also closed.
Is there a way I can prevent this behavior so that anything opened from that yazi instance remains open?

**Comment by smartding** [comment]
I have a similar problem.
I use yazi has a file picker,  after opening the file, I want to quit yazi.
But when I quit yazi, the file I just opened is also closed.

**Comment by dawsers** [comment]
Try setting `orphan = true` and see if that helps. It should detach the application from the task system so it will keep on living even if yazi exits.

**Comment by smartding** [comment, marked as answer]
I added a opener with `orphan = true` as follows, and it works.
I also see that there's an FAQ entry named [Why is "orphan" set to false by default?](https://yazi-rs.github.io/docs/faq#why-orphan-false), so this setting has side effects.
```toml
[opener]
open = [
  { run = 'xdg-open "$@"', orphan = true, desc = "Open", for = "unix" }, # add orphan = true
]
```

**Comment by dawsers** [comment]
The only side effect would be for tasks you may want to cancel, like copying or decompressing files. Those tasks usually run in the shell, so it would be harder to cancel them otherwise. If you just want to open a PDF, `orphan = true` makes sense, because if you needed to close the PDF viewer, you would usually do it from the application or the task manager.

**Comment by a-torgovitsky** [comment]
Thanks for the helpful discussion!
